libs/ui-lib/lib/ocm/components/clusterWizard/OperatorsStep.tsx

```diff
@@ -25,6 +25,7 @@ export const OperatorsStep = (props: ClusterOperatorProps) => {
       </StackItem>
       <StackItem>
         <MceCheckbox
+          {...props}
```

Comment:
If you do the spread, then you should remove `openshiftVersion` in line 30 as it's already part of props.

Alternatively, leave the version and add the clusterId prop.